src/transformers/data/metrics/__init__.py

```diff
@@ -23,12 +25,21 @@
     from scipy.stats import pearsonr, spearmanr
 
 
+DEPRECATION_WARNING = (
+    "This metric will be removed from the library soon, metrics should be handled with the 🤗 Datasets "
+    "library. You can have a look at this example script for pointers: "
+    "https://github.com/huggingface/transformers/blob/master/examples/text-classification/run_glue.py",
```

Comment:
Hi, the comma at the end of this line causes `DEPRECATION_WARNING` to be a tuple, which means `warnings.warn(DEPRECATION_WARNING, FutureWarning)` fails with `TypeError: expected string or bytes-like object`. 

I can also make a formal issue if making a comment in an already-merged PR is not the correct way to bring this to your attention (For instance, I'm not sure who, if anyone, would get notified by this comment).

Reply:
Saw it, will push a fix :-)